model_analyzer/record/metrics_manager.py

```diff
@@ -601,6 +624,12 @@ def _run_perf_analyzer(
             return (None, None)
 
         perf_records = perf_analyzer.get_perf_records()
+
+        if self._config.model_type == "LLM":
```

Comment:
I would think all of this would be handled inside of get_perf_records(). That it would look to see if LLM or not and extend what it returns.

Reply:
I thought about doing that. It's ugly no matter where this step happens. This will go away once we switch from using CSVs to an API.